src/mqt/bench/utils.py

```diff
@@ -428,3 +413,17 @@ def calc_supermarq_features(
         parallelism,
         liveness,
     )
+
+
+def get_module_for_benchmark(benchmark_name) -> ModuleType:
+    if benchmark_name in ["portfolioqaoa", "portfoliovqe", "pricingcall", "pricingput"]:
+        return import_module("mqt.bench.benchmarks.qiskit_application_finance." + benchmark_name)
+    if benchmark_name == "qgan":
+        return import_module("mqt.bench.benchmarks.qiskit_application_machine_learning.qgan")
+    if benchmark_name == "groundstate":
+        return import_module("mqt.bench.benchmarks.qiskit_application_nature.groundstate")
+    if benchmark_name == "routing":
+        return import_module("mqt.bench.benchmarks.qiskit_application_optimization.routing")
+    if benchmark_name == "tsp":
+        return import_module("mqt.bench.benchmarks.qiskit_application_optimization.tsp")
```

Comment:
would it be possible to cover all of these in tests so that we can make sure every import works?

Reply:
Very good point. Added a test which immediately revealed a wrong name in the imports.

Reply:
Glad that I looked at it 😌